packages/flutter_test/lib/src/binding.dart

```diff
@@ -665,9 +684,63 @@ class _TestViewConfiguration extends ViewConfiguration {
   String toString() => 'TestViewConfiguration';
 }
 
+const int _kPointerDecay = -2;
+
+class _LiveTestPointerRecord {
+  _LiveTestPointerRecord(
+    int pointer,
+    this.position
+  ) : pointer = pointer,
+      color = new HSVColor.fromAHSV(0.8, (35.0 * pointer) % 360.0, 1.0, 1.0).toColor(),
+      decay = 1;
+  final int pointer;
+  final Color color;
+  Point position;
+  int decay; // >0 means down, <0 means up, increases by one each time, removed at 0
+}
+
+class _LiveTestRenderView extends RenderView {
+  _LiveTestRenderView({
+    ViewConfiguration configuration
+  }) : super(configuration: configuration);
+
+  final Map<int, _LiveTestPointerRecord> _pointers = <int, _LiveTestPointerRecord>{};
+
+  @override
+  void paint(PaintingContext context, Offset offset) {
+    assert(offset == Offset.zero);
+    super.paint(context, offset);
+    if (_pointers.isNotEmpty) {
+      final double radius = configuration.size.shortestSide * 0.05;
+      final Path path = new Path()
+        ..addOval(new Rect.fromCircle(center: Point.origin, radius: radius))
+        ..moveTo(0.0, -radius * 2.0)
+        ..lineTo(0.0, radius * 2.0)
+        ..moveTo(-radius * 2.0, 0.0)
+        ..lineTo(radius * 2.0, 0.0);
+      final Canvas canvas = context.canvas;
+      final Paint paint = new Paint()
+        ..strokeWidth = radius / 10.0
+        ..style = PaintingStyle.stroke;
+      for (int pointer in _pointers.keys) {
+        _LiveTestPointerRecord record = _pointers[pointer];
+        paint.color = record.color.withOpacity(record.decay < 0 ? (record.decay / (_kPointerDecay - 1)) : 1.0);
+        canvas.drawPath(path.shift(record.position.toOffset()), paint);
+        record.decay += 1;
+        scheduleMicrotask(markNeedsPaint);
```

Comment:
Do you really want to markNeedsPaint for each pointer?  Seems like one call ought to be enough.